implementation/src/main/java/io/smallrye/openapi/runtime/scanner/AnnotationScannerExtension.java

```diff
@@ -47,4 +47,19 @@ default boolean isJaxRsAnnotationExtension(AnnotationInstance instance) {
         return false;
     }
 
+    /**
+     * Parses an OpenAPI Extension value. The value may be:
+     *
+     * - JSON object - starts with '{'
+     * - JSON array - starts with '['
+     * - number
+     * - boolean
+     * - string
+     *
+     * @param key the name of the extension property
+     * @param value the string value of the extension
+     */
+    default Object parseExtension(String key, String value) {
```

Comment:
@tjquinno, please let me know if this method definition will cover what you need to parse extensions.

Reply:
Thanks @MikeEdgar . I think the answer is "yes." I had to shift gears to some other things briefly so I'm not in a position right now to try this (probably first thing next week). But I think it looks as if it'll work.

Reply:
@MikeEdgar Just FYI, I'm using a local build of the latest SmallRye master (with this PR) and I see my `AnnotationScannerExtension` `parseExtension` method being invoked as expected. Thanks again.

Reply:
@tjquinno - very good, glad to hear that. I expect there will be a release in the next few days.